File: python/example/selenium/Pages/search/search_page.py
from selenium.webdriver.common.by import By
from citronella import ui
from Pages.components import Header, Footer


class SearchPage(Header, Footer):
    def search_lists_result(self):
        return ui(By.XPATH, '//span[@class="package-snippet__name"]')

    def order_by_option(self):
        return ui(By.ID, 'order')

File: javascript/example/page/search/search-page.js
const { By } = require("selenium-webdriver");
const { ui } = require("citronella");
const MenuBar = require("../components/menu-bar");

class SearchPage extends MenuBar {
  get searchResultLists() {
    return ui(By.css('a[target="_self"]'));
  }
}

module.exports = SearchPage;
